compiler/rustc_resolve/src/lib.rs

```diff
@@ -524,17 +524,19 @@ enum ModuleKind {
     /// * A normal module – either `mod from_file;` or `mod from_block { }` –
     ///   or the crate root (which is conceptually a top-level module).
     ///   The crate root will have `None` for the symbol.
+    ///     * the bool in the tuple next to symbol represents whether the "transparent"
+    ///     attribute is present on the module, only applies to named `mod` items.
     /// * A trait or an enum (it implicitly contains associated types, methods and variant
     ///   constructors).
-    Def(DefKind, DefId, Option<Symbol>),
+    Def(DefKind, DefId, Option<(Symbol, bool)>),
```

Comment:
This was the most obvious place to put this but it doesn't feel great, don't love all the places where I changed shit to `name_and_transparent`, open to suggestions for how to better thread this information through